dnf/db/history.py

```diff
@@ -426,67 +424,6 @@ def beg(self, rpmdb_version, using_pkgs, tsis, cmdline=None, comment=""):
         except:
             pass
 
-        '''
-        for pkg in using_pkgs:
-            pid = self.pkg2pid(pkg)
-            self.swdb.trans_with(tid, pid)
-        '''
-
-        # add RPMs to the transaction
-        # TODO: _populate_rpm_ts() ?
-
-        if self.group:
-            for group_id, group_item in sorted(self.group._installed.items()):
-                repoid = ""
-                action = libdnf.transaction.TransactionItemAction_INSTALL
-                reason = libdnf.transaction.TransactionItemReason_USER
-                replaced_by = None
-                ti = self.swdb.addItem(group_item, repoid, action, reason)
-                ti.setState(libdnf.transaction.TransactionItemState_DONE)
```

Comment:
Are you sure that the transaction also mark - ti.setState(libdnf.transaction.TransactionItemState_DONE). For RPM it is handled in dnf code but not for groups. @lukash Please could you verify it?

Reply:
Thanks